features/fakes/FakeIdBrokerClient.php

```diff
@@ -125,9 +126,8 @@ public function mfaCreate($employee_id, $type, $label = null)
      * Get a list of MFA configurations for given user
      * @param string $employee_id
      * @return array
-     * @throws ServiceException
      */
-    public function mfaList($employee_id)
+    public function mfaList(string $employee_id)
```

Comment:
Did you check all the calls to make sure everything was passing a defined value which was a string? Because debugging a type mismatch as a result of a function call is a pain to debug.

Reply:
Not thoroughly, but thanks to your reminder, I'll check more. This one's OK. The real IdBrokerClient is already strong typed and we haven't seen any errors from it on production.